docs/mast/mast.rst

```diff
@@ -181,6 +180,57 @@ To get a table of metadata associated with observation or product lists use the
 
 
 
+Mission Searches
+================
+
+Mission-Specific Search Queries
+-------------------------------
+
+These queries allow for searches based on mission-specific metadata for a given
+data collection.  Currently it provides access to a broad set of Hubble Space
+Telescope (HST) metadata, including header keywords, proposal information, and
+observational parameters.  The available metadata includes all information that
+was previously available in the original HST web search form, and are present in
+the current Mission Search interface.
+
+.. code-block:: python
+
+                >>> from astroquery.mast import Datasets
+                >>> params = {"target": ["40.66963 -0.01328"],
```

Comment:
The documentation seems a little incomplete to me.  As a small example, this code sample doesn't actually do a query.  In general as a user, I'd be looking for the following things:

- At a high level, how do the missions searches differ from the others (i.e., when might I choose to use the missions search?).
- What queries are available to me?
- What do each of the keys in the params dict do?  (And do astroquery users need to know about all the params shown here?)
- What is the syntax for specifying "conditions"?
- (A pointer to) documentation about what columns are available, along with their types so I know what conditions would be applicable.
- Can I do a follow-up query to retrieve data products?